src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/schema/metric_ops.ts

```diff
@@ -64,6 +64,7 @@ export const esqlColumnSchema = schema.object({
   /**
    * Value
    */
+  operation: schema.literal('value'),
```

Comment:
adding operation to the esql part as well, as else typescript autocompletion gets very confused.